CliClient/locales/nl_NL.po

```diff
@@ -662,17 +662,18 @@ msgid ""
 "to list tags associated with [note]. The command `tag list` can be used to "
 "list all the tags (use -l for long option)."
 msgstr ""
-"<tag-command> kan \"add\", \"remove\" of \"list\" zijn om [tag] toe te "
-"wijzen aan of te verwijderen uit [note] of om de met [tag] geassocieerde "
-"notities op te sommen. De opdracht `tag list` kan worden gebruikt om alle "
-"labels op te sommen (gebruik “-l” voor “long” optie)."
+"<tag-command> kan \"toevoegen\", \"verwijderen\" of \"weergeven\" zijn om "
+"[tag] toe te wijzen aan of te verwijderen uit [note] of om de met [tag] "
+"geassocieerde notities op te sommen. De opdracht `tag list` kan worden "
+"gebruikt om alle labels op te sommen (gebruik “-l” voor “long” optie)."
 
```

Comment:
You can't translate commands. Otherwise Joplin won't recognize the input.
If you want you can add a translation after the word in parentheses. e.g.: \"add\" (toevoegen), ...